app/views/organizations/edit.html.erb

```diff
@@ -34,17 +34,13 @@
             <% elsif google_classroom_roster_import_enabled? && current_organization.google_course_id %>
               You are currently connected to Google Classroom.
             <% else lti_launch_enabled? || google_classroom_roster_import_enabled? %>
-              <p>You are currently not connected to a Learning Management System. Connect to this Classroom via</p>
-              <%= link_to "Google Classroom",
-                google_classrooms_index_organization_path,
+              <p>You are currently not connected to a Learning Management System. Integrating GitHub Classroom
+              with your institution may unlock new capabilities and enhance your experience.</p>
```

Comment:
`may` probably isn't a great word to use here. This implies that they might integrate and have no new capabilities, which isn't true.